components/sup/src/manager/mod.rs

```diff
@@ -116,7 +116,7 @@ pub struct ManagerConfig {
     pub auto_update: bool,
     pub eventsrv_group: Option<ServiceGroup>,
     pub update_url: String,
-    pub update_channel: Option<String>,
+    pub update_channel: String,
```

Comment:
Yeah, I like this better--forces us to determine the defaults upstream and keeps the internals simpler.